exercises/practice/hello-world/.docs/instructions.append.md

```diff
@@ -1,6 +1,6 @@
 # Hints
 
-To complete this exercise, you need to implement the `hello` function.
+To complete this exercise, you need to define the variable `hello`.
```

Comment:
It isn't exactly a _variable_ either, as it cannot change, i.e. _vary_.

```suggestion
To complete this exercise, you need to define the constant `hello`.
```

Alternatives: _name_, _value_. The former might confuse; the latter conflicts with line 6.